oaknational/Oak-Web-Application
# CTC-93: update lesson overview page

## Description

Music year: 1981

- Swaps keywords and common misconception sections on the the lesson details component
- Renames video to lesson video on the lesson overview view and corresponding types and test files

## Issue(s)

Fixes #CTC-93

ACs:

- [x]  Swap the keywords and common misconceptions sections so that keywords is above common misconceptions
- [x]  Rename video to lesson video on the side navigation and the section header
- [x]  This should be for all content

## How to test

1. Go to https://deploy-preview-3031--oak-web-application.netlify.thenational.academy/teachers/programmes/chemistry-secondary-ks4-foundation-edexcel/units/atomic-structure-and-the-periodic-table/lessons/atomic-structure-negligible-electron-mass (Or any lesson)
3. You should see video has been renamed to lesson video and keywords swapped with common misconceptions

## Screenshots

How it used to look (delete if n/a):
<img width="1274" alt="Screenshot 2024-11-27 at 10 27 36" src="https://github.com/user-attachments/assets/506cdaf5-09f1-4dfe-a692-3a752ea6e4cc">
<img width="1341" alt="Screenshot 2024-11-27 at 10 27 48" src="https://github.com/user-attachments/assets/6379b3bd-eb3b-4f9f-8c85-816d550883f4">


How it should now look:
<img width="1221" alt="Screenshot 2024-11-27 at 10 24 11" src="https://github.com/user-attachments/assets/8482d660-570d-4efe-987e-39a732d4f2b2">
<img width="1247" alt="Screenshot 2024-11-27 at 10 24 31" src="https://github.com/user-attachments/assets/6d6d64b8-1244-4752-ae0a-d2b0678a014a">

## Checklist
